easybuild/easyconfigs/b/Biopython/Biopython-1.73-foss-2019a.eb

```diff
@@ -0,0 +1,52 @@
+##
+# EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2013 The Cyprus Institute
+# Authors::   Andreas Panteli <a.panteli@cyi.ac.cy>,
+#             Thekla Loizou <t.loizou@cyi.ac.cy>,
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# License::   MIT/GPL
+#
+##
+easyblock = 'PythonPackage'
+
+name = 'Biopython'
+version = '1.73'
+
+homepage = 'http://www.biopython.org'
+description = """Biopython is a set of freely available tools for biological
+ computation written in Python by an international team of developers. It is
+ a distributed collaborative effort to develop Python libraries and
+ applications which address the needs of current and future work in
+ bioinformatics. """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['http://biopython.org/DIST']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['70c5cc27dc61c23d18bb33b6d38d70edc4b926033aea3b7434737c731c94a5e0']
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [
+    ('SciPy-bundle', '2019.03'),  # for numpy
+]
+
+download_dep_fail = True
+use_pip = True
+
+# required because we're building Python packages using Intel compilers on top of Python built with GCC
+check_ldshared = True
```

Comment:
You shouldn't need this for foss

Reply:
A total oversight, thanks for pointing it out. Fixed.